manage_all.py

```diff
@@ -33,20 +33,18 @@ def main():
 
 def load_all_services():
     services = []
-    for _, _, files_names in os.walk(SERVICE_PATH):
-        services.extend(files_names)
-
-    services = [file_name[:-4] for file_name in services]
+    for _, _, file_name in os.walk(SERVICE_PATH):
+        services.extend(file_name)
+    # Remove .yml from the of service name
+    services = [service.replace('.yml', '') for service in services]
 
     return services
 
 
 def manage_services(services, manage_type):
+    cmd = ['stackanetes', '--debug', manage_type, 'service_type']
     for service_name in services:
-        cmd = ['stackanetes', '--config-dir',
-               '/etc/stackanetes', '--debug', manage_type]
-        cmd.append(service_name)
-        print cmd
+        cmd[-1] = service_name
```

Comment:
change it to cmd + [ service_name ] 
will make cmd variable const between loop iterations